packages/prettier-config/index.json

```diff
@@ -0,0 +1,6 @@
+{
+  "printWidth": 120,
+  "tabWidth": 4,
```

Comment:
By default prettier use spaces for indentation. Does this conflict with this eslint rule https://github.com/Marfeel/coding-standards/blob/master/packages/eslint-config-js/index.js#L130?

Reply:
This actually refers to: ["the number of spaces per indentation-level"](https://prettier.io/docs/en/options.html#tab-width), not tabs vs spaces. That said, I'd prefer a `tabWidth` of 2 (the default value).

Reply:
I always use my editor on half screen so I prefer 2 as well. But that is just personal preference.